src/model/invite.rs

```diff
@@ -448,5 +448,5 @@ pub enum InviteTargetType {
 enum_number!(InviteTargetType {
     Normal,
     Stream,
-    EmmbeddedApplication,
+    EmmbeddedApplication
```

Comment:
this change is unrelated
```suggestion
    EmmbeddedApplication,
```

Reply:
`cargo +nightly fmt` is a devil on Windows.